front/pages/api/w/[wId]/data_sources/[name]/tables/csv.ts

```diff
@@ -163,7 +163,7 @@ export async function handlePostTableCsvUpsertRequest(
     });
   }
   const tableId = bodyValidation.right.tableId ?? generateModelSId();
-  const slugifyName = slugify(name);
+  const slugifyName = slugify(`${name} (${tableId.substring(0, 4)})`);
```

Comment:
Ok I guess this is slightly different since it's not the notion / gdrive ID, but works for me 👍